src/ui-lib/sass/05-components/03-organisms/00-page-structure/_00-page-header.scss

```diff
@@ -1,90 +1,92 @@
 $grav-page-heading-logo-width: 250px;
+// stylelint-disable-next-line scss/dollar-variable-pattern
+$trigger-breakpoint: gravy-breakpoint(medium);
```

Comment:
Can we rename this to `$grav-page-heading-trigger-breakpoint`? Since it's no longer within a block, it is effectively global. It don't think it clashes with anything else, but having a more specific name ought to minimise the risk of that happening in future.